package.json

```diff
@@ -55,6 +55,11 @@
           "type": "boolean",
           "default": true,
           "description": "Enables or disables the Dance key bindings."
+        },
+        "dance.normalModeLineNumbersRelative": {
```

Comment:
I'd personally name that different, like `changeLineNumbers` or `changeLineNumbersStyle` with a description "Use relative line numbers in normal mode and absolute line numbers in insert mode."

Reply:
How about giving an option for the line numbers for both insert and normal mode and then do `dance.normalMode.lineNumbers` and `dance.insertMode.lineNumbers` that can be set to any of the valid options (`off`, `on`, `relative`)

Reply:
That was also one of my thoughts. Another option (like `default`) should be added to instead use the user's `editor.lineNumbers` preference.

If you do end up implementing it this way, you can disregard the comment about not setting `On`. :)

Reply:
I'll also ask you to use the current setting as the default (unless there are several opinions against this).

Reply:
I split the options and the default for normal mode is `relative`